src/entry.js

```diff
@@ -0,0 +1,36 @@
+window.jQuery = window.$ = $; 
```

Comment:
I'm not sure how this file (`src/entry.js`) got into this PR (probably something I did while trying to address merge conflicts) but please delete it as we no longer need it.